mooncake-store/src/master_service.cpp

```diff
@@ -151,6 +177,24 @@ ErrorCode MasterService::PutStart(
                    << ", error=invalid_params";
         return ErrorCode::INVALID_PARAMS;
     }
+#ifdef USE_LRU_MASTER
+    LOG(INFO) << "### LRU Update in Put() ###";
+    if(all_key_idx_map_.find(key) != all_key_idx_map_.end())
+    {
+        all_key_list_.erase(all_key_idx_map_[key]);
+        all_key_idx_map_.erase(key);
+    }
+    all_key_list_.push_front(key);
+    all_key_idx_map_[key] = all_key_list_.begin();
+    if(all_key_list_.size() >= LRU_MAX_CAPACITY)
```

Comment:
A global lru size limitation may not good for production, we'd better limit the size at per client/node level.
Since each client could have limited cache size, but the client number could be dynamic scale up/down in production.